scripts/zones/Windurst_Waters/npcs/Leepe-Hoppe.lua

```diff
@@ -49,7 +49,11 @@ function onTrigger(player,npc)
         if (player:hasKeyItem(dsp.ki.MOON_BAUBLE)) then -- Default text after acquiring moon bauble and before fighting Fenrir
             player:startEvent(845,0,1125,334);
         elseif (player:hasKeyItem(dsp.ki.WHISPER_OF_THE_MOON)) then -- First turn-in
-            player:startEvent(846,0,13399,1208,1125,0,18165,13572);
+            local availRewards = 0
+            if (not player:hasKeyItem(dsp.ki.TRAINERS_WHISTLE)) then availRewards = availRewards + 128; end -- Mount Pact
```

Comment:
Wouldn't `not player:hasKeyItem(dsp.ki.TRAINERS_WHISTLE)` mean you can only get it if you've not completed the mount quest?

Reply:
no, this means you can only get it IF you completed the quest and have the KI

Reply:
Shortly after I made the comment, i realized its a mask remove rewards. so not having the key item makes sense, yes.